.github/workflows/checkstyle.yml

```
@@ -0,0 +1,20 @@
+name: CheckStyle
+
+on:
+  push:
+    branches: [ master ]
+  pull_request: ~
+
+jobs:
+  checkstyle_job:
+    runs-on: ubuntu-latest
+    name: Run CheckStyle checks
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Check for EasyPost style compliance
```

Nit: Could change to
```suggestion
      - name: Run Checkstyle
```